pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -285,43 +286,59 @@ func testHTTPSLibrary(t *testing.T, fetchCmd []string, prefetchLibs []string) {
 	requestCmd := exec.Command(cmd[0], cmd[1:]...)
 	out, err := requestCmd.CombinedOutput()
 	require.NoErrorf(t, err, "failed to issue request via %s: %s\n%s", fetchCmd, err, string(out))
+	fetchPid := uint32(requestCmd.Process.Pid)
+	t.Logf("%s pid %d", cmd[0], fetchPid)
 
+	var allConnections []network.ConnectionStats
+	var httpKey http.Key
 	require.Eventuallyf(t, func() bool {
 		payload := getConnections(t, tr)
+		allConnections = append(allConnections, payload.Conns...)
 		for key, stats := range payload.HTTP {
+			if key.Path.Content != "/200/foobar" {
+				continue
+			}
 			req, exists := stats.Data[200]
 			if !exists {
-				continue
+				errorLogs := fmt.Sprintf("http %# v stats %# v", krpretty.Formatter(key), krpretty.Formatter(stats.Data))
+				t.Logf("%s", errorLogs)
+				t.Fail()
```

Comment:
is it reasonable to call it inside require.Eventually?

Reply:
I moved to t.Errorf() and return false, that's doing the same thing

Reply:
t.Error calls t.Fail